WordPress/gutenberg
# [RNMobile] Align Range Control (Android Slider component) to the left of it's container

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2667

## Description

This fix simply moves to the slider to the left as much as possible so that there isn't a noticeable gap between the control and the left side of the screen. 
Based on my investigation, the native implementation of the Slider control extends `AppCompatSeekBar` and it's within a container that has it's own margin/padding. I didn't see any style attributes that could be passed so that this could be left-aligned so if there's another solution that's more optimal it's more than welcomed. 

## How has this been tested?
I verified that the behavior has adjusted the Range Control on 

- [x] Columns
- [x] Cover
- [x] Latest Posts
- [x] Linear Gradient

### To test on Android

1. Add a Spacer component to the canvas. 
2. Click the settings button. 
3. Notice that the Slider control is more left-aligned. 

-----------------
### To test on iOS

1. Add a Spacer component to the canvas. 
2. Click the settings button. 
3. Notice that the Slider control is the same and no regression has occurred. 

## Screenshots 

### Left alignment of Slider

Before | After 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/94325679-3984b680-ff65-11ea-9450-ba3abe0a3a41.png" width="320"></kbd> | <kbd><img src="https://user-images.githubusercontent.com/1509205/94325677-37baf300-ff65-11ea-8acf-d83b6e1ce69d.png" width="320"></kbd>

### Thumb ripple effect of Slider
      
As seen in the above screenshots, the control could be moved a bit more to the left but I didn't do that because I started noticing that it would get cut off and the ripple effect of the draggable thumb was being cut. 

Before | After 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/94325750-81a3d900-ff65-11ea-9881-7b4a04912fce.gif" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/94325748-7d77bb80-ff65-11ea-9db2-1d9001f1f3f2.gif" width="320"></kbd>

### Column Settings Slider

Below, I checked to ensure that this change did not impact the Columns Settings Slider usage. The main change is that there is a bit less space 

<details>
<summary>Click to see screenshots</summary> 

Before | After 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/94326237-5cfd3080-ff68-11ea-9ef7-f5426d9ae943.png" width="320"></kbd>|  <kbd><img src="https://user-images.githubusercontent.com/1509205/94326242-5f5f8a80-ff68-11ea-96e1-0d67b91d3a3b.png" width="320"></kbd>

</details>     

###  Button 
<details>
<summary>Click to see screenshots</summary>

Before | After 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/94326924-31c91000-ff6d-11ea-826a-c349b47e2877.png" width="320"></kbd>  |  <kbd><img src="https://user-images.githubusercontent.com/1509205/94326921-2fff4c80-ff6d-11ea-9024-66dae9565912.png" width="320"></kbd>
</details>

### Cover
<details>
<summary>Click to see screenshots</summary>

Before | After 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/94326942-50c7a200-ff6d-11ea-8e97-f9381326c1f4.png" width="320"></kbd>   | <kbd><img src="https://user-images.githubusercontent.com/1509205/94326939-4efdde80-ff6d-11ea-96b2-308df2bf9bfd.png" width="320"></kbd>
</details>

## Types of changes
This change involves creating a separate style for each platform and adding a `margin-left` for the Android style.

## Checklist:
